tests/unit/lambdas/api_server/test_area.py

```diff
@@ -351,6 +351,32 @@ def test_put_files_info(self):
             'checksums': {'s3_etag': 'a', 'sha1': 'b', 'sha256': 'c', 'crc32c': 'd'}
         })
 
+    def test_post_file_with_valid_area(self):
```

Comment:
I like to structure tests like this:
```
lots of
setup
                                        <--- blank line
do the thing
                                        <--- blank line
checks that the thing
did the right thing
```
This helps you understand what is setup code and what is is you're testing.
This code would benefit from that.